docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/salesforce-source.md

```diff
@@ -129,6 +129,26 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Mana
 
 <a href="/files/c2c/salesforce/example.tf" target="_blank">Download example</a>
 
+## Releases
+
+### 3.X.X
+Configuration required for upgrading source from 2.X.X to 3.X.X version: 
```

Comment:
```suggestion
```

Reply:
It is required, because there will be 2 types of uses.
1. Old having c2c version 2.X.X
2. New

this steps are only for old users

Reply:
We are covering this in line number 141.